src/api/account_api.js

```diff
@@ -652,6 +655,7 @@ module.exports = {
                                         type: 'string'
                                     },
                                     identity: { $ref: 'common_api#/definitions/access_key' },
+                                    azure_log_access_keys: { $ref: 'common_api#/definitions/azure_log_access_keys' },
```

Comment:
we should not return it in account_info

Reply:
why not?

Reply:
Since we consider it as sensitive as credentials, so for the same reason we don't return secret_key in account_info, we also shouldn't return this